File: Benchmarking_Languages_Project/Python_Tests/Test13.py
from scipy import *
from numpy import *
from time import *
from fractions import *

timing = zeros(15)
for j in range(15):
	time1 = time()
	a =  ceil(random.uniform(size=400000)*1000)
	b =  ceil(random.uniform(size=400000)*1000)
	T13_1 = zeros(400000)
	for i in range(400000):
		T13_1[i]=gcd(a[i],b[i])
	timing[j]=time()-time1
print mean(timing)

File: Benchmarking_Languages_Project/Julia_Tests/Test13.jl
a = randi((100,1000),400000)
b = randi((100,1000),400000)
T13_1 = zeros(400000)
tic()
for j in 1:15
for i in 1:400000
T13_1[i]=gcd(a[i],b[i])
end
end
timing = toc()
println(timing/15)
